includes/class-sensei-settings.php

```diff
@@ -322,7 +322,7 @@ public function init_fields() {
 
 		$fields['course_archive_image_enable'] = array(
 			'name'        => __( 'Course Archive Image', 'sensei-lms' ),
-			'description' => __( 'Output the Course Image on the Course Archive Page.', 'sensei-lms' ),
+			'description' => __( 'Output the Course Image on the Course Archive Page (not applied for blocks).', 'sensei-lms' ),
```

Comment:
I wonder if it would be better to say "does not apply when using blocks"?

Reply:
Looks better to me! Updated here: https://github.com/Automattic/sensei/pull/4184/commits/4ba9a0752aba6cea50b26aa3dd89de0edcf3ccb8